index.js

```diff
@@ -0,0 +1,98 @@
+let myLibrary = [];
+const auth = document.getElementById('author');
+const title = document.getElementById('title');
+const pages = document.getElementById('pages');
+const read = document.getElementById('read');
+const btn = document.getElementById('submit');
+const table = document.getElementById('table');
+
+function getBooks() {
+  let data = localStorage.getItem('library');
+  data = JSON.parse(data);
+  return data;
+}
+function setBooks() {
+  const data = JSON.stringify(myLibrary);
+  localStorage.setItem('library', data);
+}
+
+function Book(author, title, pages, read = false) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+
+function addBookToLibrary(arr, book) {
```

Comment:
## Implement function with ES6 arrow function
- [ ] Implement this `addBookToLibrary` function with ES6 arrow function
```suggestion
const addBookToLibrary = (arr, book) => {
```